### PR TITLE
Update README.md file 

### PR DESCRIPTION
**Pull request by Rajith90**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,31 +15,31 @@ Microgateway offers you a proxy that is capable of performing security validatio
...
-
-To setup a microgateway project, a developer can choose two ways.
-
- 1. Create a microgateway project for a single API
- 1. Create a microgateway project for a group of APIs
+##### Initiating  a microgateway project
 
-The first step of setting up a microgateway project includes connecting to the API Publisher (Management Layer) and downloading the relevant API artifacts (JSON representation of the APIs). Once this step is completed it will convert the JSON representation of the APIs to Ballarina source code. The annotations that go into these source files (k8s, docker annotations, etc) are governed by a config file which the microgateway can see. These generated files can optionally be managed via source management repositories (Git).
+Once the project is initiated the default folder structure will be created for the respective folder.
+Empty api_definitions folder and definitions.yaml will be created inside the folder.API developer can add multiple open API definitions inside the
```
Comment (praminda) [inline README.md]:
```suggestion
Empty `api_definitions` folder and `definitions.yaml` will be created inside the folder.API developer can add multiple open API definitions inside the
```

File: README.md [hunk 2]
```diff
@@ -14,31 +15,31 @@ Microgateway offers you a proxy that is capable of performing security validatio
...
+api_definitions file and define endpoints and interceptors for the resources in definitions.yaml
```
Comment (praminda) [inline README.md]:
```suggestion
`api_definitions` file and define endpoints and interceptors for the resources in `definitions.yaml`
```

File: README.md [hunk 3]
```diff
@@ -14,31 +15,31 @@ Microgateway offers you a proxy that is capable of performing security validatio
...
  1. Building the microgateway project and creating a microgateway distribution.
  1. Running the microgateway distribution.
 
-##### Setting up a microgateway project
-
-To setup a microgateway project, a developer can choose two ways.
-
- 1. Create a microgateway project for a single API
- 1. Create a microgateway project for a group of APIs
+##### Initiating  a microgateway project
```
Comment (praminda) [inline README.md]:
```suggestion
##### Initiating a microgateway project
```

File: README.md [hunk 4]
```diff
@@ -50,27 +51,53 @@ Following are the set of commands included within the Microgateway.
 
 Note: Before you execute any of the commands below you need to add the path to the <micro-gw-home>/bin directory to the PATH environment variable. Ex: /home/dev/wso2am-micro-gw/bin
```
Comment (praminda) [inline README.md]: How about adding a subheading for `Running the microgateway distribution`

> Build step will generated the final microgateway distribution archive. This archive can be extracted and used to run the microgateway.

File: README.md [hunk 5]
```diff
@@ -50,27 +51,53 @@ Following are the set of commands included within the Microgateway.
...
+If the project already exists, a warning will be prompted requesting permission to override existing project.
 
 Execute `micro-gw help setup` to get more detailed information regarding the setup command.
 
 Example
 
-1. Setting up a project for a single API.
 
-    `$ micro-gw setup pizzashack-project -a PizzaShackAPI -v 1.0.0`
+    `$ micro-gw init petstore-project`
```
Comment (praminda) [inline README.md]:
```suggestion
    $ micro-gw init petstore-project
```

File: README.md (hunk 2 above)
Comment (Rajith90) [reply README.md]: There is already a sub heading called "How to run the microgateway distribution"

**Pull request merged**